services/emailer/src/namex_emailer/services/helpers.py

```diff
@@ -72,27 +69,3 @@ def get_magic_link(nr_number, email, phone):
     }
     encoded_params = urlencode(params)
     return f'{BUSINESS_REGISTRY_URL}incorporateNow/?{encoded_params}'
-
-
-@staticmethod
```

Comment:
Please add some comments to your PR description summarizing the work you've done in here, especially the removal of this section and the other code above.